rootfs/api/tests/test_pods.py

```diff
@@ -285,16 +282,21 @@ def test_container_errors(self):
         response = self.client.post(url, HTTP_AUTHORIZATION='token {}'.format(self.token))
         self.assertEqual(response.status_code, 201)
         app_id = response.data['id']
+
         # create a release so we can scale
         app = App.objects.get(id=app_id)
         user = User.objects.get(username='autotest')
         build = Build.objects.create(owner=user, app=app, image="qwerty")
+
         # create an initial release
-        Release.objects.create(version=2,
-                               owner=user,
-                               app=app,
-                               config=app.config_set.latest(),
-                               build=build)
+        Release.objects.create(
+            version=2,
+            owner=user,
+            app=app,
+            config=app.config_set.latest(),
+            build=build
+        )
```

Comment:
The value is not used, it's just to create a DB record so we have a release attached to the Build object